### PR TITLE
Win

### PR DESCRIPTION
**Pull request by stotiks**

### CODE REVIEW

File: generated/include/vnx/Endpoint.hxx [hunk 1]
```diff
@@ -4,13 +4,19 @@
 #ifndef INCLUDE_vnx_Endpoint_HXX_
 #define INCLUDE_vnx_Endpoint_HXX_
 
+#ifdef _MSC_VER
+#include <vnx_base_export.h>
```
Comment (madMAx43v3r) [inline generated/include/vnx/Endpoint.hxx:8]: where's this file?

**Comment by stotiks** [comment]
It is autogenerated with GENERATE_EXPORT_HEADER Cmake macro

File: generated/include/vnx/Endpoint.hxx (hunk 1 above)
Comment (stotiks) [reply generated/include/vnx/Endpoint.hxx:8]: It is autogenerated with GENERATE_EXPORT_HEADER Cmake macro

**Comment by madMAx43v3r** [comment]
see #2

**Pull request closed without merging**